README.md

```diff
@@ -181,32 +181,32 @@ Outside of AWS, you can [pass in your own SSH keys](https://github.com/bottleroc
 
 To enable the container, you can change the setting in user data when starting Bottlerocket, for example EC2 instance user data:
 
-```
+```toml
 [settings.host-containers.admin]
 enabled = true
 ```
 
 If Bottlerocket is already running, you can enable the admin container from the default [control container](#control-container) like this:
 
-```
+```sh
 enable-admin-container
 ```
 
 Or you can start an interactive session immediately like this:
 
-```
+```sh
 enter-admin-container
 ```
 
 If you're using a custom control container, or want to make the API calls directly, you can enable the admin container like this instead:
 
-```
+```sh
 apiclient set host-containers.admin.enabled=true
 ```
 
 Once you've enabled the admin container, you can either access it through SSH or execute commands from the control container like this:
 
-```
+```sh
 apiclient exec admin bash
```

Comment:
Same here. `exec` is highlighted since it's a keyword:

![Screen Shot 2022-10-07 at 10 23 23 AM](https://user-images.githubusercontent.com/23109390/194601670-e085aedc-5c96-4f94-b5b4-6f80ec29103c.png)